netCDF4/_netCDF4.pyx

```diff
@@ -1712,7 +1721,7 @@ references to the parent Dataset or Group.
     the parent Dataset or Group.""" 
 
     def __init__(self, filename, mode='r', clobber=True, format='NETCDF4',
```

Comment:
Some text should be added to the docstring noting that `filename` is only used to set the value of `filepath` when `memory` is specified.